MANIFEST.in

```diff
@@ -4,4 +4,5 @@ include COPYING
 include COPYING.LESSER
 include README.rst
 
+recursive-exclude tests *
```

Comment:
Are you sure it is needed when you already have `find_packages(exclude=('tests',))` in `setup.py`?

Reply:
Yes, I verified that it is required as well for the command `python setup.py sdist`

Reply:
Ok, I didn't know, thanks :-)

Reply:
I used this as a reference https://stackoverflow.com/questions/8556996/setuptools-troubles-excluding-packages-including-data-files